### ISSUE TITLE
Message Center Not Receiving Initial Response - iOS

### ISSUE THREAD

**Issue by prudolph** [apptentive/apptentive-react-native#50]
While testing the message center, I cannot get responses to show up until the app is killed and restarted. Once the app has been restarted the message center shows past messages, and will also update when new messages are sent from the Apptentive dashboard as expected. 

This seems to be isolated to iOS, as I have tested on an Android build and it works properly. I have also seen the same behavior on the Apptentive IOS SDK example project.   

My sandbox application environment settings are  :

  ```
"dependencies": {
    "apptentive-react-native": "^5.3.1",
    "react": "16.6.1",
    "react-native": "0.57.5"
  },
  "devDependencies": {
    "babel-jest": "23.6.0",
    "jest": "23.6.0",
    "metro-react-native-babel-preset": "0.49.1",
    "react-test-renderer": "16.6.1"
  },
```

I am also using Xcode Version 10.1 (10B61), targeting iOS 9.0

**Comment by weeeBox** [comment]
@prudolph, we're aware of the issue and preparing a fix which would available soon in 5.3.2 release

**Comment by prudolph** [comment]
@weeeBox  Thanks for the update!

**Comment by frankus** [comment]
We have fixed this bug in version 5.2.3 of the Apptentive iOS SDK. We don't package the iOS SDK with the React Native module, so the way to get this update is to run `pod update` from inside the `ios` directory in your project. Please let us know if you run into any problems getting updated and whether this fix works for you. 

Thanks!

**Comment by frankus** [comment]
Closing for now. Please reopen if this continues to be an issue.
